maturity.go

```diff
@@ -25,6 +27,21 @@ func (s *MaturityReport) Get(category string) *Level {
 	return nil
 }
 
+func (c *Client) GetServiceMaturityWithAlias(alias string) (*ServiceMaturity, error) {
+	var q struct {
+		Account struct {
+			Service ServiceMaturity `graphql:"service(alias:$service)"`
+		}
+	}
+	v := PayloadVariables{
+		"service": graphql.String(alias),
+	}
+	if err := c.Query(&q, v); err != nil {
+		return nil, err
```

Comment:
Is it possible to develop a failure case for this?  Would we have to find a way to get the client to return an error? Not sure of the best way to do this but tried looking into it for a bit.

Reply:
ya we'd need an alternative test client that returns an http error rather then marshaling a fixture.  And then in that test you validate the error is bubbled up.  Its alot of effort for little return on test coverage.  But if you want to do it up to you.

I'd prefer we spend cycles on fixing our test fixtures first.  Once we have a patten it should be easy to go back and add these tests for error handling later - but that pattern might tie into our new test fixture pattern.